modules/installation-generate-aws-upi.adoc

```diff
@@ -71,7 +71,7 @@ you can safely ignore this warning.
 .. Remove the files that define the control plane machines:
 +
 ----
-$ rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml
+$ rm -f openshift/99_openshift-cluster-api_master-*.yaml
```

Comment:
Per my latest test result, https://bugzilla.redhat.com/show_bug.cgi?id=1698207, I think this change need to be reverted.